spec/models/ticket_spec.rb

```diff
@@ -88,11 +88,11 @@
              user: user,
              ticket: ticket,
              quantity: 20)
-      expect(ticket.quantity_bought_by(user)).to eq(20)
+      expect(ticket.quantity_bought_by(user, 'f')).to eq(20)
```

Comment:
As I was just creating ticket_purchases in this controller `paid` still remained `false`, I think i will add tests for making payments while testing `payments.rb` for values with `paid` as `true`.